### ISSUE TITLE
Segmentation fault with nameserver usage.

### ISSUE THREAD

**Issue by Copytalky** [dlundquist/sniproxy#159]
Hi, when in the following sniproxy configuration, my sniproxy fails with a 'segmentation fault' instantly upon startup. (using latest 0.3.6 sniproxy)

config:
```
user daemon

pidfile /var/tmp/sniproxy.pid

resolver {
namserver 8.8.8.8
mode ipv4_first
}
###############################################
listen 81 {
    proto http
        source 106.61.224.124
        table http
access_log {
        # Same options as error_log
        filename /tmp/sniproxy_access.log
priority debug
}
}
###############################################
listen 444 {
    proto tls
        table https
        source 106.61.224.124
access_log {
        # Same options as error_log
        filename /tmp/sniproxy_access.log
priority debug
}
}
###############################################
listen 82 {
    proto http
        source 2001:19f0:6400:87c3::64
        table http
access_log {
        # Same options as error_log
        filename /tmp/sniproxy_access.log
priority debug
}
}
###############################################
listen 445 {
    proto tls
        table https
        source 2001:19f0:6400:77c3::64
access_log {
        # Same options as error_log
        filename /tmp/sniproxy_access.log
}
}
###############################################
listen 83 {
    proto http
        source 2001:19f0:6400:77c3::65
        table http
access_log {
        # Same options as error_log
        filename /tmp/sniproxy_access.log
priority debug
}
}
###############################################
listen 446 {
    proto tls
        table https
        source 2001:19f0:6400:77c3::65
access_log {
        # Same options as error_log
        filename /tmp/sniproxy_access.log
priority debug
}
}
##############################################

table http{
.* *:80

}
table https{
.* *:443

}
```

when I run the same configuration without 'nameserver' in resolver section of my config, the configuration allows sniproxy to run fine with no complaints.

this is on ubuntu 14.01 64bit. "uname -a Linux testbox 3.16.0-31-generic #43-Ubuntu SMP Tue Mar 10 17:37:36 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
"

If you require further data let me know.

**Comment by dlundquist** [comment]
@Copytalky are you running the 0.3.6 tag or master? This sounds very familiar: 85b3500.

**Comment by Copytalky** [comment]
I had installed via downloading the master.zip. it stated sniproxy version 0.3.6 after install, then tested the configuration and get segmentation fault error. then I tried to re-install and downloaded once again master.zip then went to install and setver.sh change the install configuration from 0.3.6 to say 0.3.5 for some reason, now whenever it installs it installs stating the sniproxy is version 0.3.5.

How do I install the 0.3.6 'tag'?

**Comment by dlundquist** [comment]
I found your problem: you have "namserver" in your configuration rather than "nameserver". I'm going to look in to why the parser is segfaulting in this case.

**Comment by dlundquist** [comment]
Fixed in 67cba1b.

**Comment by Copytalky** [comment]
yup that did it thanks!
